tools/rs-server/RsServer.cpp

```diff
@@ -96,11 +96,16 @@ struct server
             for(auto stream_profile : sensor.getStreamProfiles())
             {
                 rs2::video_stream_profile stream = stream_profile.second;
-                if(stream.format() == RS2_FORMAT_BGR8 || stream.format() == RS2_FORMAT_RGB8 || stream.format() == RS2_FORMAT_Z16 || stream.format() == RS2_FORMAT_Y8 || stream.format() == RS2_FORMAT_YUYV || stream.format() == RS2_FORMAT_UYVY)
+                if(stream.format() == RS2_FORMAT_BGR8 
```

Comment:
The switch-case should go - it cannot be maintained efficiently
I would suggest to refactor into map-like container where the pass conditions are explicitly defined,
for instance:
|  Key (Stream type)  |  Formats (one of) | Resolutions | FPS  |
|---|---|---|---|
|  RS2_INFRARED |  Y8,Y16,UYUV,RGB8  | VGA  | 6,15,30,60,90  |
|  RS2_INFRARED |  Y8,Y16,UYUV,RGB8  | HD  | 6,15  |
|  RS2_DEPTH |  Z16 | VGA,HD  | 6,15  |
  
And it will be stored in a map that will be looked up for the match
std::map <Stream, Configuration> =
{        RS2_INFRARED,      {{Y8,Y16,UYUV,RGB8}     {VGA}               {6,15,30,60,90}} }
{        RS2_INFRARED,      {{Y8,Y16,UYUV,RGB8}     {HD}                       {6,15}} }
{        RS2_DEPTH,           {{Z16}                      {VGA,HD}                {6,15,30}} }